pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -62,9 +65,10 @@ def synchronize(remote_pk, repository_pk):
     remote = RpmRemote.objects.get(pk=remote_pk)
     repository = Repository.objects.get(pk=repository_pk)
 
-    package_dupe_criteria = {'model': Package,
-                             'field_names': ['name', 'epoch', 'version', 'release', 'arch']}
-
+    package_dupe_criteria = [
```

Comment:
@fabricio-aguiar , since it no longer contains criteria for packages only, I suggest to rename it to something more general, like `dupe_criteria`.